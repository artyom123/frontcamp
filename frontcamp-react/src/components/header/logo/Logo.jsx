import React from 'react';
import { makeStyles, Link } from '@material-ui/core';

import stylesConstants from '../../../styles/constants.module.scss';

const useStyles = makeStyles({
    link: {
        color: stylesConstants.primaryRed,
    },
});

const Logo = () => {
    const classes = useStyles();

    return (
        <Link className={classes.link}>
            <strong>netflix</strong>
            roulette
        </Link>
    );
};

export default Logo;