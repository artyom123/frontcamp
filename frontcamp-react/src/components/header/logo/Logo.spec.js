import React from 'react'
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo', () => {
    const logo = shallow(<Logo />)

    it('render initial', () => {
        expect(logo).toMatchSnapshot();
    });
});
