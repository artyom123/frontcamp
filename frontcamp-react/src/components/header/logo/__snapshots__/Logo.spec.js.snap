// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render initial 1`] = `ShallowWrapper {}`;

exports[`Logo render initial 1`] = `ShallowWrapper {}`;
